README.md

```diff
@@ -105,6 +105,30 @@ logger = logging.getLogger(__name__)
 logger.info('testing123...')
 ```
 
+## Clients
+Client connects with server.py on host machine to communicate with modules. Here is an example of creating a client.
+```python
+from client import client
+client_1 = client()
+```
+
+ping(self) can be issued to ping the server to get the client's IP address and binding port.
```

Comment:
No need to include the `self` in docs - it is always assumed. Instead let's say "The `ping()` method can be..."